Hudl.Mjolnir.Attributes/CommandAttribute.cs

```diff
@@ -15,10 +15,23 @@ public sealed class CommandAttribute : Attribute
         private readonly string _breakerKey;
         private readonly string _poolKey;
         private readonly int _timeout;
+        private readonly bool _ignoreTimeout;
 
         // See Mjolnir's Command constructors.
         public CommandAttribute(string group, string isolationKey, int timeout = DefaultTimeout) : this(group, isolationKey, isolationKey, timeout) { }
 
+        public CommandAttribute(string group, string isolationKey, bool ignoreTimeout)
```

Comment:
I'll be honest, part of the reason for me doing this was to make testing a little bit easier. But I can take it out and just use an injected ConfigurationProvider, as is done in the current set of tests.